lib/mina/infinum/ecs/db.rb

```diff
@@ -26,7 +26,7 @@
         --target #{jump_server_id}
         --document-name AWS-StartPortForwardingSessionToRemoteHost
         --parameters host="#{host}",portNumber="#{remote_port}",localPortNumber="#{local_port}"
-        --profile #{profile}
+        #{"--profile #{profile}" if profile}
```

Comment:
Let's extract this to a helper in the `aws.rb` file:

```suggestion
        #{aws_cli_profile_flag}
```

We can use this helper in all the places.

Reply:
👍